corehq/apps/domain/templates/domain/partials/bootstrap3/payment_modal.html

```diff
@@ -211,7 +211,7 @@
                        size="20"
                        name="emails"
                        class="form-control"
-                       placeholder="jane@gmail.com john@gmail.com" />
+                       placeholder="jane@gmail.com, john@gmail.com" />
```

Comment:
is this user facing or accounting only? if user facing can we implement a multiselect select2 widget, or is that too complicated with a bootstrap 3 page?

Reply:
It is user-facing, though evidently it doesn't used very often and the intent is ultimately to move it off of this page altogether and into the accounting admin interface.

Frankly, I'm not sure how to implement that widget here. I see it used with crispy forms for emails, but I didn't see anywhere it was done with html/knockout like this. I tried importing `accounting/js/widgets` making the element a `<select>` and applying the `.accounting-email-select2` class (which seemed like essentially what the crispy forms versions were doing), and it did _something_ but wasn't actually usable.